requirements-dev.txt

```diff
@@ -2,3 +2,4 @@ pytest==2.8.5
 pytest-cov==2.2.0
 pytest-pep8==1.0.6
 typing==3.6.1
+-e git+https://github.com/python/mypy@005fd6b357b995f13e3d90293cabf3c8c8c2a952#egg=mypy
```

Comment:
The issue was that one of the settings you put in `mypy.ini` is available on master but not in the latest release. That's why I chose the latest commit instead.

And yes, you're right, it does indeed cause problems on Python 2. I think the easiest way would be to manually ensure that the types are correct (by running mypy locally), and then to release the version 0.3.0. After that, we can remove Python 2 support on master and add mypy to all Travis checks again.